.travis.yml

```diff
@@ -18,11 +18,12 @@ install:
 
 before_install:
   - if [ "$TRAVIS_SECURE_ENV_VARS" == "true" ]; then
-      openssl aes-256-cbc -K $encrypted_5bb5dbfd81fd_key -iv $encrypted_5bb5dbfd81fd_iv -in travis.tar.gz.enc -out travis.tar.gz -d;
+      openssl aes-256-cbc -K $encrypted_d2c20fa49002_key -iv $encrypted_d2c20fa49002_iv -in travis.tar.gz.enc -out travis.tar.gz -d;
```

Comment:
Why new key?

Reply:
Hmm when I ran `travis encrypt-file` it gave me a new command to copy-paste. Maybe they create a new key for each file you encrypt?